test/Aqua.jl

```diff
@@ -1,11 +1,12 @@
 using Aqua
 
 @testset "Aqua.jl" begin
+  ignore = isdefined(Base, :ScopedValues) ? [:ScopedValues] : []
   Aqua.test_all(
     OhMyThreads;
     # ambiguities=(exclude=[SomePackage.some_function], broken=true),
-    # stale_deps=(ignore=[:SomePackage],),
-    deps_compat=(ignore=[:Test],),
+    stale_deps=(ignore=[:ScopedValues],),
+    deps_compat=(;ignore,),
```

Comment:
This was needed because Aqua complains if we have ScopedValues in the Project.toml but not loaded. Can be removed if we drop 1.10